features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreViewModel.kt

```diff
@@ -36,32 +41,41 @@ enum class NavigationRequest {
     SendDash, ReceiveDash, None
 }
 
+enum class FilterMode {
+    All, Online, Physical, Buy, Sell, BuySell
+}
+
 @ExperimentalCoroutinesApi
 @FlowPreview
 @HiltViewModel
 class ExploreViewModel @Inject constructor(
     private val merchantDao: MerchantDao,
     private val atmDao: AtmDao,
-    private val locationUpdatesUseCase: UserLocationState
+    private val locationProvider: UserLocationState
 ) : ViewModel() {
     companion object {
         const val QUERY_DEBOUNCE_VALUE = 300L
         const val PAGE_SIZE = 100
         const val MAX_ITEMS_IN_MEMORY = 300
+        const val METERS_IN_MILE = 1609.344
+        const val METERS_IN_KILOMETER = 1000.0
```

Comment:
so internally distances will be in meters? And the user will be seeing km or miles?

Reply:
Yes, internally we should keep everything in metric because it's used in most of map and location APIs. The switch should only happen on the higher level (the ViewModel) to avoid doing conversions all over the place. Right now switching isn't implemented yet and the user sees miles.